pyxem/signals/indexation_results.py

```diff
@@ -147,6 +161,516 @@ def _get_second_best_phase(z):
+        """Convert the orientation map to an `orix.CrystalMap` object"""
+        if self.axes_manager.navigation_dimension != 2:
+            raise ValueError(
+                "Only 2D navigation supported. Please raise an issue if you are interested in "
+                "support for 3+ navigation dimensions."
+            )
+
+        x, y = [ax.axis for ax in self.axes_manager.navigation_axes]
+        xx, yy = np.meshgrid(x, y)
+        xx = xx - np.min(xx)
+        yy = yy - np.min(yy)
+        scan_unit = self.axes_manager.navigation_axes[0].units
+        rotations = self.to_rotation(flatten=True)
+        phase_index = self.to_phase_index()
+
+        if self.simulation.has_multiple_phases:
+            phases = PhaseList(list(self.simulation.phases))
+            if phase_index.ndim == 3:
+                phase_index = phase_index[..., 0]
+            phase_index = phase_index.flatten()
+        else:
+            phases = PhaseList(self.simulation.phases)
+        if scan_unit is Undefined:
+            scan_unit = "px"
+
+        return CrystalMap(
+            rotations=rotations,
+            x=xx.flatten(),
+            phase_id=phase_index,
+            y=yy.flatten(),
+            scan_unit=scan_unit,
+            phase_list=phases,
+        )
+
+    def to_ipf_markers(self):
+        """Convert the orientation map to a set of inverse pole figure
+        markers which visualizes the best matching orientations in the
+        reduced S2 space.
+        """
+        if self._lazy:
+            raise ValueError(
+                "Cannot create markers from lazy signal. Please compute the signal first."
+            )
+        if self.simulation.has_multiple_phases:
+            raise ValueError("Multiple phases found in simulation")
+
+        orients = self.to_single_phase_orientations()
+        sector = self.simulation.phases.point_group.fundamental_sector
+        labels = _get_ipf_axes_labels(
+            sector.vertices, symmetry=self.simulation.phases.point_group
+        )
+        s = StereographicProjection()
+        vectors = orients * Vector3d.zvector()
+        edges = _closed_edges_in_hemisphere(sector.edges, sector)
+        vectors = vectors.in_fundamental_sector(self.simulation.phases.point_group)
+        x, y = s.vector2xy(vectors)
+        ex, ey = s.vector2xy(edges)
+        tx, ty = s.vector2xy(sector.vertices)
+
+        x = x.reshape(vectors.shape)
+        y = y.reshape(vectors.shape)
+        cor = self.data[..., 1]
+
+        offsets = np.empty(shape=vectors.shape[:-1], dtype=object)
+        correlation = np.empty(shape=vectors.shape[:-1], dtype=object)
+        original_offset = np.vstack((ex, ey)).T
+        texts_offset = np.vstack((tx, ty)).T
+
+        mins, maxes = original_offset.min(axis=0), original_offset.max(axis=0)
+
+        original_offset = (
+            (original_offset - ((maxes + mins) / 2)) / (maxes - mins) * 0.2
+        )
+        original_offset = original_offset + 0.85
```

Comment:
There are a couple different magic numbers in this function. I have only tested on cubic data, have you tried different (larger) point groups to see if the reduced zone IPF fits correctly?

Reply:
This should work as expected but I haven't been super thourough about testing this.